lib/renderers/I18NRenderer.js

```diff
@@ -30,10 +30,14 @@ class I18NRenderer extends DefaultRenderer {
           returnObjects: true,
         });
 
+        if (_.isString(message) && message === result.reply) {
+          throw new Error(`Locale views is not implemented: ${alexaEvent.request.locale}`);
```

Comment:
I don't think there should be a default locale since the words are different. It's true US and UK are similar but what if the user program the german version, we shouldn't speak in English looking for a default language.

I can change the error message to this other one.